styles.css

```diff
@@ -0,0 +1,250 @@
+* {
+    padding:0;
+    margin:0;
+    
+}
+body {
+}
+header {
+    width: 100%;
+    height:225px;
+    background-color:navy;
+
+}
+.header-holder {
+    width:90%;
+    margin: 0px auto;
+    display:flex;
+    justify-content: space-between;
+}
+
+.left-side-header {
+   margin-top: 3%;
+   color:lightgreen;
+}
+.left-side-header img {
+    width:75%;
+}
+.hidden-header {
+    font-size: 1px;
+    visibility:hidden;
+}
+.left-side-header p {
+    font-size:1.5em;
+    font-style:italic;
+}
+.right-side-header {
+    color:white;
+    margin-top: 3%;
+}
+.shopping-cart {
+    width:20px;
+    padding-right: 2px;
+}
+.right-side-header button{
+    margin-top: 3%;
+    width:100%;
+}
+nav {
+    width:100%;
+    height:80px;
+    background-color:beige;
+
+}
+ul.header-nav-list-holder {
+    margin-left:8%;
+    height:80px;
+    line-height:80px;
+    margin-right:5px;
+    display:flex;
+    flex-direction:row;
+    list-style-type: none;
+}
+a {
+    color:navy !important;
+    text-decoration:none;
+}
+a.needs-slash::after {
+    content: "/";
+    padding-left:10px;
+    margin-right:-5px;
+}
+a.needs-slash2::after {
```

Comment:
You could probably find a way to combine these, since the padding/margins should be the same as they are in between elements